app/javascript/components/search/controls/download/DownloadCommand.jsx

```diff
@@ -44,7 +47,14 @@ export default function DownloadCommand({ fileIds=[], azulFiles }) {
       </div>
     }
     {
-      !isLoading &&
+      !isLoading && !!authCodeError &&
+      <div className="text-center text-danger">
+        <h4>There was a problem authorizing your request</h4>
+        <p dangerouslySetInnerHTML={{__html: authCodeError }} />
```

Comment:
Could you remind me why `dangerouslySetInnerHTML` is needed?

Reply:
I don't know that it is specifically, but I couldn't figure out any other way to render the error message as HTML so that the `mailto:` link works.  If there's a different/better paradigm for this I'm all ears.